website/content/docs/job-specification/parameterized.mdx

```diff
@@ -152,9 +152,59 @@ job "email-blast" {
 }
 ```
 
+### Interactions with [`periodic`][periodic] 
+
+When trying to schedule a job that is both `parameterized` and `periodic`, an internal 
+hierarchy comes into play: A parametrized job should not be dispatched without 
+parameters by the periodic configuration, forcing the parameterized option to take
+precedence: Once the job is dispatched and given parameters, the 
+periodic configuration comes into play and a new jobs will be dispatched according 
+to the periodic configurations and with the parameters of the triggering job.
+
+For example a job with the following configuration will not trigger any new jobs 
+until it is dispatched at least once, and after that the dispatched child will 
+periodically trigger more children with the given parameters. 
+
+```hcl
+  periodic {
+    crons = [
+      "*/40 * * * * * *"
+    ]
+  }
+  parameterized {
+    payload       = "required"
+    meta_required = ["dispatcher_email"]
+    meta_optional = ["pager_email"]
+  }
+```
+
+The first job `batch/periodic/parameterized` corresponds to the parameterized and
+periodic job, but it won't result in any new allocations until a `dispatch` with the 
+necessary parameters is submitted. The second job `batch/periodic` will trigger 
+the subsequent jobs `batch`. Any new dispatch will trigger a new flow of new 
+periodic jobs with the corresponding parameters. 
```

Comment:
```suggestion
There are three columns plus comments in this example output, which is for the preceding periodic, parameterized example job.

1. The `sync` job is the parameterized and periodic job. However, Nomad doesn't run the periodic job until you submit with the required parameters.
1. The `sync/dispatch-1730972650-247c6e97` job contains parameters and triggers the batch jobs, named `sync/dispatch-1730972650-247c6e97/periodic-1730972680` and `sync/dispatch-1730972650-247c6e97/periodic-1730972860`.
1. The `sync/dispatch-1730972760-f79a96e1` job assigns new parameters and triggers new batch jobs that use those parameters.
```

Style guide changes. I also tried to list in order what the example illustrates.